testcase/ninja_symlink_outputs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright 2020 Google Inc. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+mk="$@"
+
+cat <<EOF >Makefile
+l1:
+	ln -sf nil l1
+l1: .KATI_SYMLINK_OUTPUTS := l1
+
+l2:
+	ln -sf nil l2a && ln -sf nil l2b
+l2: .KATI_SYMLINK_OUTPUTS := l2a l2b
+
+f:
+	touch f
+
+l3: f
+	ln -sf f l3
+l3: .KATI_SYMLINK_OUTPUTS := l3
+EOF
+
+all="l1 l2 l3"
+if echo "${mk}" | grep kati > /dev/null; then
+  mk="${mk} --use_ninja_symlink_outputs"
+fi
+${mk} -j1 $all
+
+if [ -e ninja.sh ]; then
+  ./ninja.sh -j1 $all;
+
+  mv build.ninja kati.ninja
```

Comment:
Why do we need to move the file here?

Reply:
Cargo-culted from other tests :) removed.